Sources/ComposableArchitecture/Store.swift

```diff
@@ -7,11 +7,19 @@ import ReactiveSwift
 /// You will typically construct a single one of these at the root of your application, and then use
 /// the `scope` method to derive more focused stores that can be passed to subviews.
 public final class Store<State, Action> {
-  @MutableProperty private(set) var state: State
+  @MutableProperty
+  private(set) var state: State
+  
   private var isSending = false
   private let reducer: (inout State, Action) -> Effect<Action, Never>
   private var synchronousActionsToSend: [Action] = []
   private var bufferedActions: [Action] = []
+  
+  @DisposablesScopedStorage
```

Comment:
What is the benefit of using a property wrapper instead of just a plain dictionary?

Reply:
You can have a `deinit` in `Store` which does the same as the `disposeAll` logic above.

Reply:
Well, no actual profit, will be removed

Reply:
Fixed